socorro/tests/external/test_crashstorage_base.py

```diff
@@ -150,204 +145,6 @@ def test_basic_crashstorage(self):
-            "B.crashstorage_class": "socorro.tests.external.test_crashstorage_base.B",
-            "A2.y": 37,
-        }
-        cm = ConfigurationManager(n, values_source_list=[value])
-        with cm.context() as config:
-            assert config.A.crashstorage_class.foo == "a"
-            assert config.A2.crashstorage_class.foo == "a"
-            assert config.A2.y == 37
-            assert config.B.crashstorage_class.foo == "b"
-
-            poly_store = config.storage(config)
-            assert len(poly_store.storage_namespaces) == 3
-            assert poly_store.storage_namespaces[0] == "A"
-            assert poly_store.storage_namespaces[1] == "A2"
-            assert poly_store.storage_namespaces[2] == "B"
-
-            assert len(poly_store.stores) == 3
-            assert poly_store.stores.A.foo == "a"
-            assert poly_store.stores.A2.foo == "a"
-            assert poly_store.stores.B.foo == "b"
-
-            raw_crash = {"ooid": ""}
-            dump = "12345"
-            processed_crash = {"ooid": "", "product": 17}
-            for v in poly_store.stores.values():
-                v.save_raw_crash = mock.Mock()
-                v.save_processed_crash = mock.Mock()
-                v.close = mock.Mock()
-
-            poly_store.save_raw_crash(raw_crash, dump, "")
-            for v in poly_store.stores.values():
-                v.save_raw_crash.assert_called_once_with(raw_crash, dump, "")
-
-            poly_store.save_processed_crash(raw_crash, processed_crash)
-            for v in poly_store.stores.values():
-                v.save_processed_crash.assert_called_once_with(
-                    raw_crash, processed_crash
-                )
-
-            raw_crash = {"ooid": "oaeu"}
-            dump = "5432"
-            processed_crash = {"ooid": "aoeu", "product": 33}
-
-            expected = Exception("this is messed up")
-            poly_store.stores["A2"].save_raw_crash = mock.Mock()
-            poly_store.stores["A2"].save_raw_crash.side_effect = expected
-            poly_store.stores["B"].save_processed_crash = mock.Mock()
-            poly_store.stores["B"].save_processed_crash.side_effect = expected
-
-            with pytest.raises(PolyStorageError):
-                poly_store.save_raw_crash(raw_crash, dump, "")
-
-            for v in poly_store.stores.values():
-                v.save_raw_crash.assert_called_with(raw_crash, dump, "")
-
-            with pytest.raises(PolyStorageError):
-                poly_store.save_processed_crash(raw_crash, processed_crash)
-
-            for v in poly_store.stores.values():
-                v.save_processed_crash.assert_called_with(raw_crash, processed_crash)
-
-            poly_store.stores["B"].close.side_effect = Exception
-            with pytest.raises(PolyStorageError):
-                poly_store.close()
-
-            for v in poly_store.stores.values():
-                v.close.assert_called_with()
-
-
-class TestBench:
```

Comment:
We only use these benchmarking wrappers to capture the `save_processed_crash` timings. Instead of having benchmarking wrappers, we're going to add timings capture code to the processor so we're capturing the things we want to capture there without the need of wrapper classes.